contracts/mock/MockApi3Pool.sol

```diff
@@ -1,4 +1,21 @@
 //SPDX-License-Identifier: MIT
```

Comment:
I may revisit these mock contracts or even use the actual ones later

Reply:
Yeah just to refresh your memory, we agreed that after the contract work that you would look to update the [api3-dao/blob/main/packages/dao/scripts/deploy.js](https://github.com/api3dao/api3-dao/blob/main/packages/dao/scripts/deploy.js) script to deploy these claims-manager contracts as well, as we will need the ClaimsManager to work with the actual Api3Pool. The ClaimsManager needs to be whitelisted by the Api3Pool: https://github.com/api3dao/api3-dao/blob/main/packages/pool/contracts/StateUtils.sol#L283

Reply:
Noted